src/types/accessors/AbstractMetabolicModel.jl

```diff
@@ -141,6 +141,25 @@ groups collect these sets of enzymes for convenient analysis.
 """
 )
 
+@make_variable_semantics(
+    :metabolite_log_concentration,
+    "metabolite log concentration",
+    """
+Certain model types use metabolite concentrations instead of reaction fluxes are
+variables. This semantic grouping uses the log (base e) metabolite concentration
+to make thermodynamic calculations easier.
+"""
+)
+
+@make_variable_semantics(
+    :gibbs_free_energy_reaction,
```

Comment:
this is a good dodge of the "energys". We should really have `gibbs_free_energy` and `gibbs_free_energies`. I'll make a macro update that fixes this.

Reply:
actually in this case you don't need to, because the value calculated is literally the Gibbs free energy of reaction

Reply:
yeah it would be nice to group all the semantics somewhere for easy reference